src/cts/src/CtsOptions.h

```diff
@@ -209,20 +207,19 @@ class CtsOptions
   std::string treeBuffer_ = "";
   std::string metricFile_ = "";
   int dbUnits_ = -1;
-  unsigned wireSegmentUnit_ = 0;
+  unsigned wireSegmentUnitDbu_ = 0;
   bool plotSolution_ = false;
   bool simpleCts_ = false;
   bool sinkClusteringEnable_ = true;
   bool sinkClusteringUseMaxCap_ = true;
   bool simpleSegmentsEnable_ = false;
   bool vertexBuffersEnable_ = false;
   bool gui_debug_ = false;
-  double vertexBufDistance_ = 240;
-  double bufDistance_ = 100;
+  double vertexBufDistance_ = 0; // in DBU 
+  double bufDistance_ = 0; // in DBU
```

Comment:
Comment is DBU but type is double? Also why change value to 0?

Reply:
These two entries used to be in Microns; therefore, they were double, will convert them to ints. I changed the value to 0 because it was set to a value that could be considered as default; however, it was not really used at all as these values are only used when a boolean is activated that depends on these values being passed in the tcl.